script/hassfest/manifest.py

```diff
@@ -13,6 +13,7 @@
     vol.Required('documentation'): str,
     vol.Required('requirements'): [str],
     vol.Required('dependencies'): [str],
+    vol.Optional('after_dependencies'): [str],
```

Comment:
```suggestion
    vol.Optional('after_dependencies', default=[]): [str],
```

Reply:
This is not needed. The code that uses `after_dependencies` will check for this.

(on a sidenote, we should avoid mutable default values, and use `default=list` instead to generate one when needed)